pkg/scheduler/framework/plugins/default_registry.go

```diff
@@ -255,6 +259,12 @@ func NewDefaultConfigProducerRegistry() *ConfigProducerRegistry {
 			return
 		})
 
+	registry.RegisterPriority(serviceaffinity.Name,
```

Comment:
If you remove this, I believe the tests wouldn't pass since only a single priority is actually registered as a prioritizer. This is important for us as a way to revert back to the old scheduling path in case of emergencies.

I think we should change the existing priorities to accept lists as we discussed before, the priority would divide by the number of labels. We still add up the weights to compensate for that.